### PR TITLE
change to rino docker containers

### LINKED ISSUE #969: melo.tools is migrating to community.rino.io
We need to migrate the default remote node and the docker images for the CI tests.

https://www.reddit.com/r/Monero/comments/t770no/migration_of_melotools_to_communityrinoio/

### PR DESCRIPTION
**Pull request by lescuer97**
this is for updating from the melo tools dockers to the rino ones

Fixes #969

### CODE REVIEW

**Comment by binarybaron** [comment]
Thanks! Can you also update the default remote node?

https://github.com/comit-network/xmr-btc-swap/blob/637574af438dca7d22e0c9c6cd9a90e6f082f55e/swap/src/cli/command.rs#L18-L19

**Comment by lescuer97** [comment]
thank you for noticing!

**Comment by binarybaron** [comment]
We should probably prepare a release after this gets merged. All other versions (including the latest release) will probably start breaking within a week or two when the old melo.tools domain goes down.

**Comment by lescuer97** [comment]
Yeah, I agree

File: CHANGELOG.md [hunk 1]
```diff
@@ -305,7 +306,7 @@ It is possible to migrate critical data from the old db to the sqlite but there
 - Fixed an issue where Alice would not verify if Bob's Bitcoin lock transaction is semantically correct, i.e. pays the agreed upon amount to an output owned by both of them.
   Fixing this required a **breaking change** on the network layer and hence old versions are not compatible with this version.
 
-[Unreleased]: https://github.com/comit-network/xmr-btc-swap/compare/0.10.2...HEAD
+[unreleased]: https://github.com/comit-network/xmr-btc-swap/compare/0.10.2...HEAD
```
Comment (da-kami) [inline CHANGELOG.md:309]: Is this change on purpose? Was the change triggered by formatter? 🤔

**Review by da-kami — APPROVED** [review]
Did you test the new URLs by starting CLI  / ASB?

**Comment by da-kami** [comment]
nit: Would be nice to link the announcement next time so I can check that it's real.
I guess this should do :)
https://www.reddit.com/r/Monero/comments/t770no/migration_of_melotools_to_communityrinoio/

File: CHANGELOG.md (hunk 1 above)
Comment (lescuer97) [reply CHANGELOG.md:309]: d print changed this

**Comment by lescuer97** [comment]
> nit: Would be nice to link the announcement next time so I can check that it's real. I guess this should do :) https://www.reddit.com/r/Monero/comments/t770no/migration_of_melotools_to_communityrinoio/

It's actually on the linked issue that is why I didn't bother

File: monero-harness/src/image.rs [hunk 2]
```diff
@@ -25,7 +25,7 @@ impl Image for Monerod {
     type EntryPoint = str;
 
     fn descriptor(&self) -> String {
-        "melotools/monero:v0.17.2.0".to_owned()
+        "rinocommunity/monero:v0.17.2.0".to_owned()
```
Comment (lescuer97) [inline monero-harness/src/image.rs:28]: This need to be changed to 17.3.0 because there is no tag for 17.2.0 Rinocommunity

File: monero-harness/src/image.rs [hunk 3]
```diff
@@ -78,7 +78,7 @@ impl Image for MoneroWalletRpc {
...
```
Comment (lescuer97) [inline monero-harness/src/image.rs:81]: This need to be changed to 17.3.0 because there is no tag for 17.2.0 Rinocommunity

File: monero-harness/src/image.rs (hunk 2 above)
Comment (delta1) [reply monero-harness/src/image.rs:28]: Yes I'd suggest trying 0.17.3 to see if the CI passes, since a patch version upgrade should not be a breaking change

**Comment by binarybaron** [comment]
Can we prepare a new minor release once this is merged?

**Comment by thomaseizinger** [comment]
Nit: The commit message should start with an uppercase letter. Have a look at the blogpost linked in the contribution guidelines :)

https://github.com/comit-network/xmr-btc-swap/blob/master/CONTRIBUTING.md

**Comment by thomaseizinger** [comment]
Otherwise looks good.

bors d=lescuer97
